.gitignore

```
@@ -42,6 +43,8 @@
 *.pem*
 *.sql*
 *.sqlite3*
+**/.terraform/*
```

```suggestion
.terraform
```

It's the same thing, but simpler.

---

Okies 👍